b2/console_tool.py

```diff
@@ -344,6 +344,25 @@ def run(self, args):
         return 0
 
 
+class GetFileInfoAndDeleteFileVersion(Command):
```

Comment:
I think the custom logic can be contained within `DeleteFileVersion.parse_arg_list` like this:
```python
OPTIONAL = ['fileName', 'fileId']
REQUIRED = []
def parse_arg_list(self, *args, **kwargs):
    args = super(DeleteFileVersion, self).parse_arg_list(*args, **kwargs)
    if args.fileId is None and args.fileName is None:
        return None
    elif args.fileName is None:  # swap the miss-assigned arguments back to their place
        args.fileId = args.fileName
        args.fileName = None
    return args
```